admin/services/retro-input-assistant.service.js

```diff
@@ -63,6 +64,20 @@ const service = {
       fullName: `${p.lastName}, ${p.foreName}${p.middleNames ? ' ' + p.middleNames : ''}`,
       urlSlug: p.urlSlug
     }))
+  },
+  /**
+ * Returns pupils with eligible for access arrangements
```

Comment:
This comment and the name of the service called, `deleteRetroInputAssistant` suggests that one of them may be incorrect!

Reply:
comment aligned, good spot!